core_context_impl/src/main/java/com/google/instrumentation/trace/GrpcTraceUtils.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2017, Google Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.instrumentation.trace;
+
+import io.grpc.Context;
+
+/**
+ * Util methods/functionality for gRPC system.
+ */
+public final class GrpcTraceUtils {
+  // Reserved {@link io.grpc.Context.Key} for the {@link Span}.
```

Comment:
Done. I think the agreement is to not use javadocs for private things. in the IDE you can click on the next line directly on the type.